packages/react/src/tests/e2e.test.tsx

```diff
@@ -84,13 +85,44 @@ describe('@remote-ui/react', () => {
     expect(appElement.innerHTML).toBe(`<div>Hello, ${name}</div>`);
   });
 
+  it('renders component with fragment as prop across a remote bridge', () => {
+    const name = 'Winston';
+
+    const receiver = createRemoteReceiver();
+    const remoteRoot = createRemoteRoot(receiver.receive, {
+      components: ['HelloWorld'],
+    });
+
+    function RemoteApp() {
+      return <RemoteHelloWorld name={<RemoteHelloWorld name={name} />} />;
+    }
+
+    const controller = createController({HelloWorld: HostHelloWorld});
+
+    function HostApp() {
+      return <RemoteRenderer controller={controller} receiver={receiver} />;
+    }
+
+    domAct(() => {
+      domRender(<HostApp />, appElement);
+      render(<RemoteApp />, remoteRoot, () => {
+        remoteRoot.mount();
+      });
+      jest.runAllTimers();
+    });
+
+    expect(appElement.innerHTML).toBe(
+      `<div>Hello, <div>Hello, ${name}</div></div>`,
+    );
+  });
```

Comment:
Do you think it's worth testing any of the more complicated cases that we discovered didn't work in the last implementation?￼